### DISCUSSION TITLE
Why some brackets have space inside, but some not?

### DISCUSSION THREAD

**Discussion by GrigoriiPrudnikov** [zed-industries/zed#15021] [Feature Requests]
I've opened outline panel and saw this
<img width="239" alt="Screenshot 2024-07-23 at 21 54 52" src="https://github.com/user-attachments/assets/70de4ea8-2afa-43e2-b989-4eb9fa799089">

Why some brackets have space inside, and some not?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
